src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/ModellingGroupTests.kt

```diff
@@ -36,6 +37,65 @@ class ModellingGroupTests : DatabaseTest()
         }
     }
 
+    @Test
+    fun `can get modelling group memberships for user`()
+    {
+        JooqContext().use {
+            it.addGroup("a", "description a")
+            it.addGroup("b", "description b")
+            it.addGroup("c", "description c")
+
+            setupTestUser()
+        }
+
+        val permissions = PermissionSet(
```

Comment:
This is actually a case where I would use the DSL, as this test doesn't currently check whether the route is secured (whereas validate.. requringPermissions... does that automatically.)